storage/raft.go

```diff
@@ -32,8 +33,16 @@ type committedCommand struct {
 	cmd      proto.InternalRaftCommand
 }
 
-// raft is the interface exposed by a raft implementation.
-type raft interface {
+// raftInterface is the interface exposed by a raft implementation.
+type raftInterface interface {
+	// createGroup initializes a raft group with the given id.
+	createGroup(int64) error
+
+	removeGroup(int64) error
```

Comment:
Added comment to removeGroup. With the bootstrapping changes, createGroup and restoreGroup are now identical so I've removed restoreGroup.